CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 Changelog
 =========
 
+## [9.2.0](https://github.com/ckeditor/ckeditor5-react/compare/v9.1.0...v10.0.0) (2024-09-19)
+
+### Features
+
+* **Experimental:** Improve result type of `useCKEditorCloud` to make `CKEditorPremiumFeatures` and `CKBox` non-nullable when proper configuration is passed. ([commit](https://github.com/ckeditor/ckeditor5-react/commit/1268d9efeeb5e34ee4f93e0e4d5e4e449ea59603))
+
+### Bug fixes
+
+* **Experimental:** Rename the `languages` configuration property to `translations` in `useCKEditorCloud`. ([commit](https://github.com/ckeditor/ckeditor5-react/commit/1268d9efeeb5e34ee4f93e0e4d5e4e449ea59603))
```

Comment:
I am unsure if it's a fix or _Other change_ as it looks like "aligning to a dependency API" instead of "fixing something because we made a mistake".

Reply:
@pomek renamed